html-forms/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+    <form action=" " method="post">
+
+      <div class="enter-email">
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input type="text" name="email">
+      </div>
+
+      <div class="codingcoolbc">
+        <label for="codingcoolbc">Why is coding cool?</label>
+      </div>
+      <div>
+        <select name="codingcoolbc">
+          <option value="selected">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+
+      <div class="favlang">
+        <label for="favlang">Favorite Language (choose one)</label>
```

Comment:
only thing to update here is to have a label for each radio input. you can wrap them like this if you prefer
```html
<label>
    <input type="radio" name="favoriteLanguage" value="HTML">
    HTML
 </label>